tests/data/bench_runner.toml

```diff
@@ -25,3 +25,6 @@ markers = ["s", "s", "s", "^", ".", "."]
 
 [publish_mirror]
 skip = false
+
+[comparisons_diff_flags]
+Base_default_vs_PR_NOGIL = false
```

Comment:
Can we move this under the existing section `[bases]` and make it configurable like `compare_to_default = ["NOGIL"]`?

Reply:
And let's document it in the template, something like:

```
A list of configuration flags that are compared against the default build of its commit merge base.
```